content/en/_index.html

```diff
@@ -63,85 +65,49 @@ <h2 class="home-section-title home-section-title-projects">Projects</h2>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/kitex">
             <h2>Kitex</h2>
-            <p>Next generation high performance, highly scalable Golang RPC framework.</p>
+            <p>A next-gen Golang RPC framework transforming microservices with unprecedented scalability and performance.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/hertz">
             <h2>Hertz</h2>
-            <p>Next generation high performance, highly scalable Golang HTTP framework.</p>
+            <p>Next-gen, high-performance Golang HTTP framework simplifying microservices development.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/netpoll">
             <h2>Netpoll</h2>
-            <p>High-performance NIO (Non-blocking I/O) network library, focusing on RPC scenarios.</p>
+            <p>A high-performance NIO network library, dedicated to building reliable, efficient services with a focus on RPC scenarios.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/thriftgo">
-            <h2>Thriftgo</h2>
-            <p>A thrift compiler implemented by Golang supports plug-in mechanism.</p>
+            <h2>Volo</h2>
+            <p>High Performance RPC Framework for high-performance & extensibility, in Rust.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/cloudwego/volo">
-            <h2>Volo</h2>
-            <p>A high-performance and strong-extensibility Rust RPC framework.</p>
+            <h2>Cwgo</h2>
+            <p>All-in-one code Generation for efficient development.</p>
         </a>
     </div>
     <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
         <a href="https://github.com/bytedance/monoio">
-            <h2>Monoio</h2>
-            <p>A thread-per-core Rust runtime with io_uring/epoll/kqueue.</p>
-        </a>
-    </div>
-    <div class="col-md-3 col-sm-6 col-xs-12 home-project-item">
-        <a href="https://github.com/cloudwego/motore">
             <h2>Motore</h2>
-            <p>Async middleware abstraction powered by AFIT and RPITIT.</p>
+            <p>Empowering extensibility with async middleware abstraction, powered by AFIT and RPITIT.</p>
         </a>
     </div>
```

Comment:
This is not correct, Motore != Monoio

Reply:
Please double check, Monoio was removed. This is the description for Motore, based on the previous Motore description @GuangmingLuo

Reply:
Sorry! Thanks for the review, I fixed the links with the update below @GuangmingLuo